basic-examples/expressions/README.md

```diff
@@ -2,16 +2,78 @@
 
 ## Workflow Description
 
-This workflow demonstrates example expressions as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/).
+This workflow demonstrates some example expressions as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/). Arcaflow expressions are a way to reference paths within the workflow or their fields, and/or manipulate the data.

```

Comment:
The comma is superfluous.

However, rather than removing the comma, I suggest expanding what follows it into a full, independent clause.  E.g.,
> Arcaflow expressions are a way to reference workflow objects or their fields, and, within expressions, built-in functions can be used to manipulate the data.

Reply:
I don't think it's superfluous because there is a preceding 'or' in the sentence.

Reply:
It's superfluous (and incorrect...) because it separates "manipulate" from "a way to":  what we really have is "a way to reference ... or manipulate", and the two parts of the "or" go together.  If what followed the "or" were a separate sentence, then the comma would be required; if it were a dependent clause with its own verb, then the comma might be appropriate; but, it's wholly dependent on and part of the phrase preceding the comma (in fact, it's an evil "split infinitive", sharing the "to" with "reference"), so the comma is incorrect.

Reply:
We are talking technical documentation here, not perfect English grammar. It isn't that important to me, but I don't think than any use of "and/or" would be considered grammatically correct, and yet it's of common use in technical documentation.

If the sentence were all 'and' conjunctions:

Reply:
Unless using proper grammar is actually impeding understanding or obscuring meaning, I think we should use it in our documentation (and in our reports and other formal communication).  Otherwise, we're just undermining ourselves.